include/schema/common/footer.json

```diff
@@ -4,6 +4,16 @@
     "description": "Page footer configurations",
     "type": "object",
     "properties": {
+        "copyright": {
+            "$ref": "/misc/copyright.json",
```

Comment:
You do not need to include `/misc/copyright.json` here since the schema is defined in place rather than in another file.